src/test/java/com/aerospike/mapper/BooleanTest.java

```diff
@@ -0,0 +1,106 @@
+package com.aerospike.mapper;
+
+import com.aerospike.client.Key;
+import com.aerospike.client.Record;
+import com.aerospike.client.policy.Policy;
+import com.aerospike.mapper.annotations.AerospikeKey;
+import com.aerospike.mapper.annotations.AerospikeRecord;
+import com.aerospike.mapper.tools.AeroMapper;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static com.aerospike.client.Value.UseBoolBin;
+
+public class BooleanTest extends AeroMapperBaseTest {
```

Comment:
Consider adding an `@AfterAll` method to return `UseBoolBin` to its default state.